_bioinfcomputing/access_methods.md

```diff
@@ -1,5 +1,210 @@
+
+After installing XQuartz just start the app. The XQuartz icon will appear in the toolbar showing that it is running. XQuartz is now running an X11 client and that is the only interaction that you need to do with XQuartz. After XQuartz is running, open the Mac terminal. The latest version of XQuartz sets the DISPLAY environment with the default Mac terminal window. We do not recommend using the terminal that comes with XQuartz.
+
+When connecting to a `rhino` with `ssh` always use the -X flag to forward your Xsession to your Mac.
+
+```
+ssh -X HutchID@rhino2
+```
+
+After connecting to a `rhino` verify that your X11 client is working by typing `xeyes`.
+
+
+### Linux
+
+Terminator is a convenient terminal emulator that can have multiple terminals in a single window. You can use Terminator on your Linux desktop. Windows and Mac users will need a X11 software to use SciComp resources as graphical desktop. We recommend [NoMachine NX](https://teams.fhcrc.org/sites/citwiki/SciComp/Pages/NX%20Client%20for%20Windows.aspx) to connect to a Linux machine with Terminator.  
+
+After starting Terminator you can right click on the terminal window and either split it horizontally or vertically. This 1 min video shows you how it works:
+
+[![Terminator](https://img.youtube.com/vi/OmkkbvH_JZU/0.jpg)](https://youtu.be/OmkkbvH_JZU "See how Terminator works")
+
+
+
+## SSH Connections
+SSH uses the TCP network protocol to make connections between systems. TCP is a persistent, stateful connection that is resilient to network interruptions. When you use SSH to run a program on a remote system (even a shell) the connection you make is tied to an SSH process on the remote machine, and all programs are children of that SSH process. If that process stops, all children stop as well. If the network connection does drop between your client and the server, your SSH process stops, and all your running programs stop.
+
+There are several ways around this:
+
+* NoMachine - you can use the NoMachine service
+* GNU Screen or tmux - these utilities run a separate process on the remote host that is not tied to your SSH connection
+* Cluster jobs - you can run your programs using our [slurm cluster](/bioinfcomputing/compute_jobs/), which queues and executes programs independently of your SSH connection
+
+### NoMachine NX (Multi-OS)
+NoMachine NX is a remote desktop software for Linux servers that gives you full remote access to a graphical user interface from Windows or Mac clients. It is installed on the SciComp session Servers `lynx`, `sphinx` or `manx` which you use to access the `rhino` systems. To connect through the local Fred Hutch network or through a VPN connection please download and install the NoMachine Enterprise Client from [NoMachine's Site.](https://www.nomachine.com/download-enterprise#NoMachine-Enterprise-Client)  Windows users can also install an older version from "Software Center".
```

Comment:
I still am not fully sure what lynx/sphinx/manx ARE, and I'm sure the reader doesn't yet know that either. Perhaps a link to content describing what they are?

Reply:
I can't think of anywhere in the wiki that we have currently have documentation on them-- maybe that's more appropriate as an issue? Would it work to just add a sentence about what a session/session server is?

Reply:
Yeah, I think this is a @dirkpetersen sort of future issue thing.  Let's not have it hold up the merge but yes, some sort of sentence describing generally what they are would be great.